uniref-rest/src/main/resources/application.properties

```diff
@@ -36,8 +36,9 @@ voldemort.unref.storeName=uniref
 
 
 ############################### Entry Store (Voldemort) Streaming properties ###############################
-streamer.uniref.batchSize=100
-streamer.uniref.valueId=id
-#streamer.uniref.defaultsField=avro_bin
+streamer.uniref.searchBatchSize=5
```

Comment:
is this batch size correct?

Reply:
i've increased it now.

but the main way to find out the correct size is by doing multiple runs of downloading and adjusting to get best performance.